### PR TITLE
WT-10805 Always display the top five latencies from the many-dhandle-test

### PR DESCRIPTION
**Pull request by Siddhartha8899**

### CODE REVIEW

**Review by ajmorton — APPROVED** [review]
LGTM 👍

File: bench/perf_run_py/perf_stat.py [hunk 1]
```diff
@@ -153,12 +153,11 @@ def get_value_list(self, brief: bool):
...
     def get_value(self, nth_max: int):
         """Return the nth maximum number from all the gathered values"""
-        return sorted(self.values)[-nth_max]
+        return sorted(self.values)[-nth_max] if nth_max < len(self.values) else 0
 
     def get_value_list(self, brief: bool):
         as_list = []
-        num_max = min(len(self.values), 5)
-        for i in range(1, num_max + 1):
+        for i in range(1, 6):
```
Comment (jiechenbo) [inline bench/perf_run_py/perf_stat.py:160]: In the case that there is 0 values, then shouldn't the list be zero? And does the old code achieve the same with the new code?

Comment (Siddhartha8899) [reply bench/perf_run_py/perf_stat.py:160]: If we have 0 values:

- Before these changes, we would not print the top 5 latencies. 
- After these changes, we will print the top 5 latencies, and they will be 0.

Refer to @ajmorton's comment in the ticket to find out why this is necessary.

Comment (Siddhartha8899) [reply bench/perf_run_py/perf_stat.py:160]: "My preference is to report these stats, as it's a useful data point in our charts to see when no latency warnings are reported. We should also review other aggregate stats to see if they need to handle the same case"

Comment (ajmorton) [reply bench/perf_run_py/perf_stat.py:160]: > We should also review other aggregate stats to see if they need to handle the same case
Just to double check @Siddhartha8899 did we find any other aggregate stats that may need the same treatment? My assumption is no as our other stats tend to be much larger numbers compared to numLatencyWarnings

Comment (Siddhartha8899) [reply bench/perf_run_py/perf_stat.py:160]: Yes your assumption is correct @ajmorton

Comment (jiechenbo) [reply bench/perf_run_py/perf_stat.py:160]: Thanks for explanation 🍔

**Review by jiechenbo — APPROVED** [review]
LGTM!

**Pull request merged**